upup/pkg/fi/cloudup/awstasks/route.go

```diff
@@ -33,9 +33,10 @@ type Route struct {
 	Name      *string
 	Lifecycle *fi.Lifecycle
 
-	RouteTable *RouteTable
-	Instance   *Instance
-	CIDR       *string
+	RouteTable           *RouteTable
+	Instance             *Instance
```

Comment:
This is a private API, so you might be able to get away with making Instance a `*string`, which might be easier (?).

But thank you for cleaning up the `Instance` task - I think it is currently unused, so it makes sense it would be a little broken!